lib/Models/WebProcessingServiceCatalogFunction.js

```diff
@@ -177,6 +223,60 @@ WebProcessingServiceCatalogFunction.parameterConverters = [
         }
     },
     {
+         id: 'RegionGeometry',
+         inputToFunctionParameter: function(catalogFunction, input) {
+             if (!defined(input.ComplexData) || !defined(input.ComplexData.Default) || !defined(input.ComplexData.Default.Format) || !defined(input.ComplexData.Default.Format.Schema)) {
+                 return undefined;
+             }
+
+             var schema = input.ComplexData.Default.Format.Schema;
+             if (schema.indexOf('http://geojson.org/geojson-spec.html#') !== 0) {
+                 return undefined;
+             }
+
+             if (schema.substring(schema.lastIndexOf('#') + 1) !== 'multipolygon') {
+                 return undefined;
+             }
+
+             if (input.Identifier.indexOf('region') === -1) {
+                 return undefined;
+             }
+
+             if (!defined(catalogFunction.regionTypeParameter)) {
+                 throw new DeveloperError('No regionTypeParameter found, but regionGeometry requires a regionTypeParameter.');
+             }
+             catalogFunction.regionTypeParameter.showInUi = false;
+             return new RegionParameter({
+                 terria: catalogFunction.terria,
+                 catalogFunction: catalogFunction,
+                 id: input.Identifier,
+                 name: input.Title,
+                 description: input.Abstract,
+                 isRequired: (input.minOccurs | 0) > 0,
+                 regionProvider: catalogFunction.regionTypeParameter
+             });
+         },
+         functionParameterToInput: function(catalogFunction, parameter, value) {
+             if (!defined(value) || value === '' || !defined(value.id)) {
+                 return undefined;
+             }
+
+             // Value is an object with id and fid. We want an actual polygon for WPS, but not for analytics.
```

Comment:
I'd argue that's because the input is a polygon, and we're only specifying the polygon by selecting a region as a convenience.  It's possible a different kind of input (even if it's one we don't have yet) could accept a region type and identifier.